sdks/java/core/src/main/java/org/apache/beam/sdk/metrics/CounterCell.java

```diff
@@ -57,28 +57,11 @@ public Long getCumulative() {
     return value.get();
   }
 
-  @Override
-  public Counter getInterface() {
-    return this;
-  }
-
-  @Override
   public void inc() {
     add(1);
   }
 
-  @Override
   public void inc(long n) {
     add(n);
   }
-
-  @Override
-  public void dec() {
```

Comment:
So counters can never be decremented? Doesn't seem unreasonable, and may make it easier for backends to aggregate, but this seems like an orthogonal PR.